.github/workflows/test-package.yml

```diff
@@ -59,8 +59,12 @@ jobs:
       - name: Setup Flutter
         run: |
           cd $HOME
-          git clone https://github.com/flutter/flutter.git --depth 1 -b ${{ matrix.flutter }} _flutter
-          echo "$HOME/_flutter/bin" >> $GITHUB_PATH
+          echo "${{ matrix.os }}"
+          RELEASE_OS=$(if [ "${{ matrix.os }}" = "macos-latest" ]; then echo "macos"; else echo "linux"; fi)
+          curl -sS "https://storage.googleapis.com/flutter_infra_release/releases/stable/${RELEASE_OS}/flutter_${RELEASE_OS}_${{ matrix.flutter }}-stable.tar.xz" >flutter.tar.xz
+          tar xf flutter.tar.xz
+          rm flutter.tar.xz
+          echo "$HOME/flutter/bin" >> $GITHUB_PATH
           cd $GITHUB_WORKSPACE
```

Comment:
Could we just use: https://github.com/flutter-actions/setup-flutter or `subosito/flutter-action`

pinned to a particular commit obviously,

Reply:
Done.